src/ddmd/mars.d

```diff
@@ -186,67 +187,6 @@ Where:
 ", FileName.canonicalName(global.inifilename), fpic, m32mscoff, mscrtlib);
 }
 
-/// DMD-generated module `__entrypoint` where the C main resides
-extern (C++) __gshared Module entrypoint = null;
-/// Module in which the D main is
-extern (C++) __gshared Module rootHasMain = null;
```

Comment:
This does more than that, but if the PR passes I will close the bug report manually and link the PR